src/components/network/network.js

```diff
@@ -271,4 +252,93 @@ export default class Network {
             this.getThreeWindingsTransformer(id)
         );
     }
+
+    generateEquipementHandler({ errHandler, postUpdate, ...equipment }) {
+        for (const [key, value] of Object.entries(equipment)) {
+            console.log(key, value, postUpdate);
+            this.lazyLoaders.set(
+                key,
+                new RemoteResourceHandler(
+                    value,
+                    (value) =>
+                        this.dispatch(updateNetwork(key, value, postUpdate)),
+                    (key) => errHandler(key)
+                )
+            );
+        }
+    }
+
+    fetchEquipment(equipement, cb) {
```

Comment:
why is the parameter called "cb" (I guess for callback?), but it's passed to fetch() which takes a boolean forceUpdate ?